tests/CMakeLists.txt

```diff
@@ -204,11 +131,14 @@ set_target_properties(mock_process
   RUNTIME_OUTPUT_DIRECTORY_DEBUG "${CMAKE_BINARY_DIR}/bin/mocks"
   RUNTIME_OUTPUT_DIRECTORY_RELEASE "${CMAKE_BINARY_DIR}/bin/mocks")
 
-add_executable(apparmor_parser
-  mock_apparmor_parser.cpp)
+add_dependencies(multipass_tests mock_process)
+
+target_include_directories(multipass_tests
+  BEFORE
+    PRIVATE ${CMAKE_SOURCE_DIR}/src/platform/backends/shared/linux
+)
```

Comment:
Why is this needed?

Reply:
This replaces `PLATFORM_INCLUDES` above. Otherwise it matters where I put `add_subdirectories`.

Reply:
Ah, alright.